eventdata/parameter_sources/elasticlogs_kibana_source.py

```diff
@@ -34,28 +34,31 @@ class ElasticlogsKibanaSource:
+        "query_string"          -   String indicating file to load or list of strings indicating actual query parameters to randomize during benchmarking. Defaults 
+                                    to ["*"], If a list has been specified, a random value will be selected.
+        "index_pattern"         -   String or list of strings representing the index pattern to query. Defaults to 'elasticlogs-*'. If a list has 
+                                    been specified, a random value will be selected.
+        "window_end"            -   Specification of aggregation window end or period within which it should end. If one single value is specified, 
+                                    that will be used to anchor the window. If two values are given in a comma separated list, the end of the window
+                                    will be randomized within this interval. Values can be either absolute or relative:
+                                        'now' - Always evaluated to the current timestamp. This is the default value.
+                                        'now-1h' - Offset to the current timestamp. Consists of a number and either m (minutes), h (hours) or d (days).
+                                        '2016-12-20 20:12:32' - Exact timestamp.
+                                        'START' - If fieldstats has been run for the index pattern and `@timestamp` field, 'START' can be used to reference the start of this interval.
+                                        'END' - If fieldstats has been run for the index pattern and `@timestamp` field, 'END' can be used to reference the end of this interval.
+                                        'END-40%' - When an interval has been specified based on fieldstats, it is possible to express a volume
+                                        relative to the size of the interval as a percentage. If we assume the interval covers 10 hours, 'END-40%'
+                                        represents the timestamp 4 hours (40% of the 10 hour interval) before the END timestamp.
+        "window_length"         -   String indicating length of the time window to aggregate across. Values can be either absolute 
+                                    or relative. Defaults to '1d'.
+                                        '4d' - Consists of a number and either m (minutes), h (hours) or d (days). Can not be lower than 1 minute.
+                                        '10%' - Length given as percentage of window size. Only available when fieldstats_id have been specified.
+        "timeout"               -   Request timeout in milliseconds. Defaults to 60000.
+        "discover_size"         -   Nunmber of documents to return in Discover. Defaults to 500.
+        "ignore_frozen"         -   Boolean indicating whether frozen indices should be queried. Defaults to `true`.
```

Comment:
The docs should probably say: "... indicating whether frozen indices should be **ignored**". Also, double-negatives can be hard to grasp.

Reply:
Seems to not have completed this properly. Sorry about that.